.coveragerc

```diff
@@ -0,0 +1,12 @@
+[run]
+source = zope.security
```

Comment:
Personally I think it's fine add `omit = */tests/*` to measure the coverage of the actual code, without having to pepper nocover pragmas for all the unused stubs/helpers in my tests.

Reply:
I have held that opinion at times too. However, someone (I think maybe @tseaver) insisted that it was worth measuring test coverage too...and sure enough, not long after that I found a project with a whole bunch of tests that just weren't even being run. If test coverage was disabled, that wouldn't have happened.

I think the stubs issue isn't really that bad, at least not in my experience so far. It's actually a good thing to change a `def never_called(): pass` into a `def never_called(): raise AssertionError("why am I being called?")` to help make tests more explicit. And if there are bunches of unused/unimported files sitting around (like the case here)...well, why are they even there? It just adds extra overhead.